crates/libm-cdylib/src/test_utils.rs

```diff
@@ -0,0 +1,143 @@
+use std::{fs, io, path::Path, process};
```

Comment:
I personally feel like this file may be a bit over-the-top when testing, I'm not sure it's really all that necessary to be invoking Cargo from this crate's own test suite to test various C things? 

If it's really this extensive then I would prefer this not live in this repository to help keep this repository manageable and not pull in a whole set of new maintenance.

Reply:
It isn't expensive on my 8 year old system - `cargo` is invoked on every test, but once the first invocation completes, all of them just return "success - the library was already built". 

I can use a `lazy_static!` to just call `cargo` once, but it felt like unnecessary complexity.